Add a piece of text describing that you can have more than one DS record (#32)
PowerDNS/parent-signals-dot: draft-vandijk-dprive-ds-dot-signal-and-pin/draft-vandijk-dprive-ds-dot-signal-and-pin.md

It MUST use the hashes attached to the DS records with DNSKEY algorithm type TBD to check whether the public key supplied by the authoritative nameserver is valid.
If the DoT connection is unsuccessful or the public key supplied the server does not match one of the DS digests, the resolver MUST NOT fall back to unencrypted Do53.

A domain MAY have more than one DS record with DNSKEY algorithm TBD.

> A resolver with support for TBD should then try to verify the public key supplied by the authoritative nameserver against all supplied DS records.

Reviewer: ```suggestion
A resolver with support for TBD should then try to verify the public key supplied by the authoritative nameserver against every supplied DS record.
```

before people think that every DS should match every key

> Multiple records can be used to support, multiple hash algorithms, have a different key for each authoritative, for key rollovers or supporting multiple certificate algorithms.

Reviewer: ```suggestion
Multiple records can be used to support multiple DS digest types, multiple TLS key algorithms, different keys for each authoritative, and for key rollovers.
```

> In case of an algorithm rollover the DS record should be added to all served domains before the new key is deployed on the authoritatives.

Reviewer: ```suggestion
In case of an algorithm or key rollover the new DS record should be added to all served domains before the new key is deployed on the authoritatives.
```

> To allow for emergency rollovers having a standby DS record for all domains with a private key securely stored offline can be a valid strategy.

Reviewer: ```suggestion
To allow for emergency rollovers, having a standby DS record for all domains with a private key securely stored offline can be a valid strategy.
```

  - Algorithm: TBD
  - Digest Type: the current Digest Type we are computing the DS for.
  - Digest: Following [@!RFC4034] section 5.1.4, compute the digest of owner name | previously computed DNSKEY's RDATA.

> 6. Test the computed DS record against all the supplied DS records untill a match is encountered.

Reviewer: ```suggestion
6. Test the computed DS record against all the supplied DS records until a match is encountered.
```
